create-rust-app_cli/src/content/project.rs

```diff
@@ -381,7 +381,7 @@ pub fn create(project_name: &str, creation_options: CreationOptions) -> Result<(
 
         logger::add_file_msg(filename.as_ref());
         std::fs::create_dir_all(directory_path)?;
-        std::fs::write(file_path, file_contents)?;
+        std::fs::write(file_path, file_contents.data)?;
```

Comment:
This change was made to resolve issues caused by updating rust-embed from "5.9.0" to "6.3.0", which was important because the old version had several security vulnerabilities